rmgpy/rmg/main.py

```diff
@@ -689,9 +697,26 @@ def execute(self, **kwargs):
         coreSpec, coreReac, edgeSpec, edgeReac = self.reactionModel.getModelSize()
         logging.info('The final model core has %s species and %s reactions' % (coreSpec, coreReac))
         logging.info('The final model edge has %s species and %s reactions' % (edgeSpec, edgeReac))
-        
         self.finish()
 
+    def generateCanteraFiles(self, chemkinFile, **kwargs):
+        """
```

Comment:
I know it's probably not straightforward to write a unit test for this, but have you tested locally?